rule.go

```diff
@@ -395,9 +404,12 @@ func (r *Rule) Run(ctx context.Context, sess *session.Session, noWait bool) erro
 			command = append(command, aws.String(v))
 		}
 		contaierOverrides = append(contaierOverrides, &ecs.ContainerOverride{
-			Name:        aws.String(co.Name),
-			Environment: kvPairs,
-			Command:     command,
+			Name:              aws.String(co.Name),
+			Environment:       kvPairs,
+			Command:           command,
+			Cpu:               aws.Int64(co.Cpu),
```

Comment:
Shouldn't we take care of the case where CPU, Memory and MemoryReservation are zero?
When there is a zero value in there, it seemed more appropriate to pass nil in this case for backward compatibility.

Reply:
I see,
Is this OK? [80686a5](https://github.com/Songmu/ecschedule/pull/62/commits/80686a5e4102bdee2fa1392e427dd50e84097e42)